### PR TITLE
Feature/fro 54 document UI

### PR DESCRIPTION
**Pull request by PatFawbertMills**
Apologies this got too big and I should've separated earlier.

- Considerable updates to the document viewer page
- Updates to the search results UI including slide out (does not contain the upcoming change to the drawer + pdf passages)
- A lot of formatting updates

### CODE REVIEW

**Comment by PatFawbertMills** [comment]
Also includes fixing a few random bugs like Justine's height issue on her small laptop screen.

File: frontend/src/components/blocks/DocumentInfo.tsx [hunk 1]
```diff
@@ -13,29 +14,25 @@ interface DoucmentInfoProps {
...
             <Tooltip id={id} tooltip={tooltip} />
           </div>
         )}
       </h4>
-      {list.length ? (
-        <List list={list} />
-      ) : (
-        <p className="text-indigo-500">{text}</p>
-      )}
+      {/* {list.length ? <List list={list} /> : <p className="text-indigo-500">{text}</p>} */}
```
Comment (diversemix) [inline frontend/src/components/blocks/DocumentInfo.tsx:32]: Commented out code - intentional?

File: frontend/src/components/headers/DocumentSlideout.tsx [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
+import TextLink from "../nav/TextLink";
+
+import { convertDate } from "@utils/timedate";
+
+const DocumentSlideout = ({ document, showPDF, setShowPDF, setPassageIndex }) => {
+
+  if (!document) return null;
+
+  const formatDate = () => {
+    const eudate = document?.document_date;
```
Comment (diversemix) [inline frontend/src/components/headers/DocumentSlideout.tsx:12]: Are we getting a guaranteed date format here?

File: frontend/src/components/blocks/SearchResult.tsx [hunk 3]
```diff
@@ -56,30 +42,20 @@ const SearchResult = ({ document }: SearchResultProps) => {
...
       </div>
 
       <div className="flex flex-wrap lg:flex-nowrap text-sm text-indigo-400 mt-4 items-center">
-        <div
-          className={`rounded-sm border border-black flag-icon-background flag-icon-${document.document_country_code.toLowerCase()}`}
-        />
+        <div className={`rounded-sm border border-black flag-icon-background flag-icon-${document.document_country_code.toLowerCase()}`} />
         <span className="ml-2">
-          {document.document_country_english_shortname}
+          {document.document_country_english_shortname}, {year}
```
Comment (joel-wright) [inline frontend/src/components/blocks/SearchResult.tsx:53]: Does this also need to be labelled as a future candidate for translation as above?

File: frontend/src/pages/document/[docId].tsx [hunk 4]
```diff
@@ -1,80 +1,83 @@
...
-      toggleSummary();
-    }
-  }, [page, showFullSummary]);
-
-  useEffect(() => {
-    if (router?.query.docId) {
-      documentQuery.refetch();
-    }
-  }, [router.query]);
+  // TODO: align with BE on an approach to sources and their logos
```
Comment (joel-wright) [inline frontend/src/pages/document/[docId].tsx:79]: I wonder how we link the addition of a new data source, the creation of a new source in the database, the storage of a logo, ... something to talk about I guess :)

**Review by joel-wright — APPROVED** [review]
lgtm

File: frontend/src/components/blocks/SearchResult.tsx (hunk 3 above)
Comment (PatFawbertMills) [reply frontend/src/components/blocks/SearchResult.tsx:53]: Yes true, dynamic translations need to be considered. So far we have some limited static translations for basic copy but things like countries and other taxonomies are just rendered as English.

**Pull request merged**